### DISCUSSION TITLE
使用表情包功能

### DISCUSSION THREAD

**Discussion by Calkull** [azmiao/uma_plugin#64] [Q&A]
用的马娘整合版，在群里使用马娘表情包时一直没反应。看了一下后台日志是这样的
![ROORZCZ}SVNH9U7Z5%%X6LA](https://github.com/azmiao/uma_plugin/assets/115563704/1fac4069-1134-4476-b1e0-6d67af747d78)

**Comment by azmiao** [comment]
看一下hoshinoBot/res/img/umamusume/文件夹下有啥东西

**Comment by Calkull** [comment]
里面是空的，大佬这正常吗？





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月6日(星期一) 下午3:34
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [azmiao/uma_plugin] 使用表情包功能 (Discussion #64)






看一下日野机器人/reses/img/umamusume/文件夹下有啥东西

-
直接回复这封邮件，在GitHub上查看，或取消订阅.
***@***.***和&gt;

**Reply by azmiao** [reply]
不正常呢，正常的话，首次启动的时候会自动更新数据的，你试试把umamusume文件夹删掉，然后重启Bot，应该会更新数据吧，你看看后台有没有报错

**Comment by Calkull** [comment]
看了一下，这里好像有问题






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "azmiao/uma_plugin"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月6日(星期一) 下午3:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [azmiao/uma_plugin] 使用表情包功能 (Discussion #64)





 
看一下hoshinoBot/res/img/umamusume/文件夹下有啥东西

**Comment by Calkull** [comment]
可以了，找到原因了。谢谢大佬了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月6日(星期一) 下午4:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [azmiao/uma_plugin] 使用表情包功能 (Discussion #64)





 
不正常呢，正常的话，首次启动的时候会自动更新数据的，你试试把umamusume文件夹删掉，然后重启Bot，应该会更新数据吧，你看看后台有没有报错

**Reply by azmiao** [reply]
好的
